src/percetto.cc

```diff
@@ -561,6 +561,7 @@ int percetto_init(size_t category_count,
 
   perfetto::TracingInitArgs args;
   args.backends = perfetto::kSystemBackend;
+  args.shmem_size_hint_kb = 32 * 1024;
```

Comment:
This is a bit small for a default - okay for now though. I guess this should be configurable in percetto_init.

Reply:
I simply copied what is done in mesa, maybe it should be part of the config file too, and not be specified in the source code.